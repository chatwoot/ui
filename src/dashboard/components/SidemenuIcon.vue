<template>
  <i class="ion-android-menu hamburger--menu" @click="onMenuItemClick" />
</template>

<script>
export default {
  methods: {
    onMenuItemClick() {
      bus.$emit('sidemenu_icon_click');
    },
  },
};
</script>
<style lang="scss" scoped>
.ion-android-menu {
  font-size: var(--font-size-big);
}
</style>
