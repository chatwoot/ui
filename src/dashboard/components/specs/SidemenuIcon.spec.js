import { mount } from '@vue/test-utils';
import SidemenuIcon from '../SidemenuIcon';

describe('SidemenuIcon', () => {
  test('matches snapshot', () => {
    const wrapper = mount(SidemenuIcon);
    expect(wrapper.vm).toBeTruthy();
    expect(wrapper.element).toMatchSnapshot();
  });
});
