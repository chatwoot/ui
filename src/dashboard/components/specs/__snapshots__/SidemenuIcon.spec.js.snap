// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidemenuIcon matches snapshot 1`] = `
<i
  class="ion-android-menu hamburger--menu"
/>
`;
